ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master.yaml

```diff
@@ -92,7 +92,10 @@ tests:
     test:
     - as: e2e-pao-mixed-cpus
       cli: latest
-      commands: CLUSTER=mcp-only make cluster-label-worker-cnf cluster-deploy-pao
+      commands: |
```

Comment:
Let's do this in different patches, step registry in one patch and these jobs in a separate.
It's different areas with different jobs and permissions, better to keep them separated.